pkgs/tools/package-management/nix/default.nix

```diff
@@ -110,9 +110,7 @@ let
         outputsToInstall = [ "out" "man" ];
       };
 
-      passthru = {
-        inherit fromGit;
-
+      passthru = { inherit fromGit; } // (if includesPerl then { perl-bindings = nix; } else {
         perl-bindings = stdenv.mkDerivation {
```

Comment:
Could push the conditional in and maybe make this a bit nicer?

```nix
{
  // ...
  passthru = {
    inherit fromGut;
    perl-bindings = if includesPerl then nix else stdenv.mkDerivation {
      // ...
    };
  };
  // ...
}
```

Reply:
done.